src/corefx/System.Globalization.Native/timeZoneInfo.cpp

```diff
@@ -22,3 +27,36 @@ extern "C" int32_t ReadLink(const char* path, char* result, size_t resultCapacit
     result[r] = '\0';
     return true;
 }
+
+/*
+* These values should be kept in sync with the managed Interop.GlobalizationInterop.TimeZoneDisplayNameType enum.
+*/
+enum TimeZoneDisplayNameType : int32_t
+{
+    Generic = 0,
+    Standard = 1,
+    DaylightSavings = 2,
+};
+
+/*
+Gets the localized display name for the specified time zone.
+*/
+extern "C" ResultCode GetTimeZoneDisplayName(
```

Comment:
I believe @jkotas gave everything in the shim prefixes as we have in corefx.  We should do the same with these new methods.